float32 rate
---
int16 status